client/src/views/page-components/ProductDetailPage/ProductDetail/ProductDetailBody/ProductDetailBody.js

```diff
@@ -0,0 +1,40 @@
+import { createDOMwithSelector } from '../../../../../utils/createDOMwithSelector.js';
+
+export default function ProductDetailBody({ $target, initialState }) {
+  this.state = initialState;
+  this.$productDetailBody = createDOMwithSelector('div', '.productDetailBody');
+  $target.appendChild(this.$productDetailBody);
+
+  this.render = () => {
+    this.$productDetailBody.innerHTML = `
+      <div class="productDetailBody__title__container">
+        <h2 class="productDetailBody__title">${this.state.title}</h2>
+        <ul class="productDetailBdoy__subInfo">
+          <li class="subInfo__title">${this.state.category}</li>
+          <li class="subInfo__title">${this.state.time}</li>
+        </ul>
+      </div>
+      <textarea class="productDetailBody__content" readonly>${this.state.content}</textarea>
+      <div class="productDetailBody__info__container">
+        <ul class="productDetailBody__info">
+          <li class="productDetailBody__info_subtitle">채팅 ${this.state.chatCounts}</li>
+          <li class="productDetailBody__info_subtitle">관심 ${this.state.likeCounts}</li>
+          <li class="productDetailBody__info_subtitle">조회 ${this.state.viewCounts}</li>
+        </ul>
+        <div class="productDetailBody__sellerInfo">
+          <div class="sellerInfo__title__container">
+            <span class="sellerInfo__title">판매자 정보</span>
+          </div>
+          <div class="sellerInfo__name__container">
+            <span class="sellerInfo__name">${this.state.seller}</span>
+            <span class="sellerInfo__town">${this.state.town}</span>
+          </div>
+        </div>
+      </div>`;
+    if (this.state.isSeller) {
+      this.$productDetailBody.classList.add('sellerIsTrue');
```

Comment:
seller is true는 문장 같네요 :) 정확히 어떤 클래스인지는 모르겠지만 눈치껏 써봤습니다.
제가 너무 깐깐한가요 🤣

```suggestion
      this.$productDetailBody.classList.add('only-shown-seller');
```

Reply:
아니요! 네이밍이 너무 깔끔하시네요!!
참고하여 수정할게요! 감사합니다 :)